BasicCrudOperations/Controllers/CourseController.cs

```diff
@@ -15,34 +16,60 @@ public CourseController(ICourseService courseService)
             _courseService = courseService;
         }
 
-        [HttpGet("GetCourses")]
+        [HttpGet("Courses")]
         public IActionResult Get()
         {
-            return Ok(_courseService.GetCourses());
+            var courses = _courseService.GetCourses();
```

Comment:
When getting all entities, you will never have `null`, you can have an empty list, but that is not a problem, you will just return `Ok` with an empty list